src/main/java/persistence/sql/entity/EntityLoader.java

```diff
@@ -0,0 +1,23 @@
+package persistence.sql.entity;
+
+import jdbc.JdbcTemplate;
+import persistence.sql.dml.SelectQueryBuilder;
+
+import java.sql.Connection;
+
+public class EntityLoader {
+    private final JdbcTemplate jdbcTemplate;
```

Comment:
`SelectQueryBuilder`도 EntityLoader 내부 상태로 가지고 있도록 구현해보면 어떨까요?
매번 엔티티를 로드할때마다 생성하지 않아도 될 것 같아요 :)

```suggestion
public class EntityLoader {
    private final JdbcTemplate jdbcTemplate;
    private final SelectQueryBuilder selectQueryBuilder;
```

Reply:
EntityPersister쪽도 마찬가지로 호출시마다 쿼리빌더를 계속 생성하고있군요. 수정해보겠습니다.